pandas/tests/reshape/merge/test_multi.py

```diff
@@ -636,12 +636,6 @@ def test_join_multi_levels_invalid(self, portfolio, household):
         ):
             household.join(portfolio, how="inner")
 
-        portfolio2 = portfolio.copy()
-        portfolio2.index.set_names(["household_id", "foo"])
-
-        with pytest.raises(ValueError, match="columns overlap but no suffix specified"):
```

Comment:
does this work now?

Reply:
yes, the join is working but since the tests merged `portfolio` with itself there are duplicated columns and now it isn't invalid

Reply:
since join is also changed and does not have the combined `suffixes` parameter and has `lsuffix` and `rsuffix` instead, can you add another (separate) release note giving a summary of the change.

Reply:
added

Reply:
@simonjayhawkins should now that join can run with empty string for right and left suffix, should I change the default value of those arguments to `_x` and `_y` (same as in merge)?